tests/unit/swagger/templateHelpers.spec.ts

```diff
@@ -246,7 +246,7 @@ describe('ValidationService', () => {
       };
       const result = v.validateModel({ name: '', value: {}, refName: 'ExampleModel', fieldErrors: error, minimalSwaggerConfig });
       expect(Object.keys(error)).to.be.empty;
-      expect(result).to.eql({ a: undefined });
```

Comment:
I was nervous about this part. It looks to me like this is a breaking change. But I think it's an acceptable change (especially since it's associated with a big fix). But I'd like @WoH to confirm if he agrees that this is an acceptable change to fall within a minor version release.

Reply:
Yeah just wanted to update the PR description about it, please double check if this does not break anything. It shouldn't and I this is actually the "new" expected behaviour if this PR will be approved ;)

Reply:
Yea, the only way to know if it breaks consumers is to solicit more input from other users. So besides @WoH, I'll also tag @lukeautry, @endor, @rgripper, @atomicus, @egrando.

Reply:
Viewing this PR, I double checked our code and found effectively a small issue with a check on a boolean value in a partial patch endpoint. Easily fixed in our case with a `=== undefined` instead of `'isActive' in` because we don't need to do the distinction. But yet, the current behavior is completely unexpected and I see no case where you would use the current behavior.

Reply:
sounds good to me, basically means in best case we can even fix some unknown bugs for some.. ;)